amanuensis/config-default.yaml

```diff
@@ -150,6 +150,10 @@ SUPPORT_EMAIL_FOR_ERRORS: null
 #   - Support `/data` endpoints
 # //////////////////////////////////////////////////////////////////////////////////////
 AWS_CREDENTIALS:
+  'AWS_SES':
```

Comment:
is the plan to have two different aws acesss keys for aws operations in amanuensis? cause if not we can just add the region name to 'DATA_DELIVERY_S3_BUCKET' and/or change the name to something more generic. If we end up keeping both the aws portion in amanuenisis/init will have to be updated to handle that

Reply:
also AWS_SES should be the second one if we dont change anything